openslides/locale/angular-gettext/template-en.pot

```diff
@@ -301,146 +627,518 @@ msgstr ""
 msgid "Permissions:"
 msgstr ""
 
-#: users/static/templates/users/user-detail.html:12
+#: users/static/templates/users/user-detail.html:25
 msgid "Personal data"
 msgstr ""
 
-#: users/static/templates/users/user-csv-import.html:12
+#: agenda/static/templates/agenda/item-import.html:34
+#: users/static/templates/users/user-import.html:34
 msgid "Please note:"
 msgstr ""
 
-#: users/static/templates/users/user-list.html:71
+#: assignments/static/templates/assignments/assignment-list.html:37
+msgid "Posts"
+msgstr ""
+
+#: motions/static/templates/motions/category-form.html:13
+#: motions/static/templates/motions/category-list.html:25
+msgid "Prefix"
+msgstr ""
+
+#: motions/static/templates/motions/category-detail.html:11
+msgid "Prefix:"
+msgstr ""
+
+#: users/static/templates/users/user-list.html:97
 msgid "Present"
 msgstr ""
 
-#: users/static/templates/users/user-csv-import.html:23
+#: agenda/static/templates/agenda/item-import.html:54
+#: users/static/templates/users/user-import.html:58
+msgid "Preview"
+msgstr ""
+
+#: agenda/static/templates/agenda/item-list.html:45
+msgid "Project agenda"
+msgstr ""
+
+#: core/static/templates/core/projector-controls.html:76
+msgid "Project countdown"
+msgstr ""
+
+#: assignments/static/templates/assignments/assignment-detail.html:16
+#: assignments/static/templates/assignments/assignment-list.html:64
+msgid "Project election"
+msgstr ""
+
+#: agenda/static/templates/agenda/item-detail.html:12
+#: agenda/static/templates/agenda/item-list.html:96
+msgid "Project item"
+msgstr ""
+
+#: core/static/templates/core/projector-controls.html:144
+msgid "Project message"
+msgstr ""
+
+#: motions/static/templates/motions/motion-detail.html:24
+#: motions/static/templates/motions/motion-list.html:100
+msgid "Project motion"
+msgstr ""
+
+#: core/static/templates/core/customslide-list.html:34
+#: users/static/templates/users/user-detail.html:12
+#: users/static/templates/users/user-list.html:112
+msgid "Project user"
+msgstr ""
+
+#: core/static/templates/core/projector-controls.html:2
```

Comment:
Please fix in template `core/static/templates/core/projector-controls.html`, line 2: The closing tag has to be a `</h3>` tag. Then the msgstr here should be updated.